### PR TITLE
Little tweaks

### PR DESCRIPTION
**Pull request by g1st**
### Proposal
I added a few tweaks to fix issues which were annoying me for some time.

### Checklist

- [x] I have made this pull request to the `master` branch
- [x] I have run `npm run lint` and there are no errors

### CODE REVIEW

File: client/src/components/CreateJob/SelectBranch.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const SelectBranch = ({ state, setState, error }) => {
 		getBranches(state.customer_id, authorizationHeaders)
 			.then((res) => {
 				setData({
-					name: "branches",
```
Comment (GTabala) [inline client/src/components/CreateJob/SelectBranch.js:19]: Why did you change this? Do you use this name in the UI?

Comment (g1st) [reply client/src/components/CreateJob/SelectBranch.js:19]: Yes it is used in the header title when address is being selected in job creation page.
